Makefile

```diff
@@ -670,6 +670,72 @@ enwiki_tuning_reports: \
 	tuning_reports/enwiki.damaging.md \
 	tuning_reports/enwiki.goodfaith.md
 
+############################# English Wiktionary ################################
+
+datasets/enwiktionary.sampled_revisions.200k_2016.json:
+	wget -qO- https://quarry.wmflabs.org/run/97131/output/0/json-lines?download=true > $@
+
+datasets/enwiktionary.autolabeled_revisions.200k_2016.json: \
+		datasets/enwiktionary.sampled_revisions.200k_2016.json
+	cat $< | \
+	./utility autolabel --host=https://en.wiktionary.org \
+		--trusted-groups=sysop,oversight,bot,rollbacker,checkuser,abusefilter,bureaucrat \
+		--trusted-edits=1000 \
+		--revert-radius=3 \
+		--revert-window=48 \
+		--verbose > $@
+
+datasets/enwiktionary.revisions_for_review.5k_2016.json: \
+		datasets/enwiktionary.autolabeled_revisions.200k_2016.json
+	grep '"needs_review": true' $< | shuf -n 5000 > $@
+
+datasets/enwiktionary.autolabeled_revisions.w_cache.20k_2016.json: \
+		datasets/enwiktionary.autolabeled_revisions.200k_2016.json
+	shuf -n 20000 $< | \
+	revscoring extract \
+		editquality.feature_lists.enwiktionary.reverted \
+		--host https://en.wiktionary.org \
+		--extractor $(max_extractors) \
+		--verbose > $@
```

Comment:
I see now that we're not training on the 200k observations at all.  Why did we sample 200k again?  I can't figure it out.  It seems like we should just use 92k observations for everything and call it good.  Is there something else that I am missing?

Reply:
I doubled checked and it seems you're not missing anything.